.circleci/continue_config.yml

```diff
@@ -207,10 +207,25 @@ jobs:
           name: Run tests
           command: make engine-docker-test
           no_output_timeout: 15m
+  
+  trigger_private_tests:
+      docker: 
+        - image: cimg/base:2021.11
+      resource_class: small
+      steps:
+        - run:
+            name: Trigger private tests
+            command: |
+              curl --request POST \
+                --url  $TOBIKO_PRIVATE_CIRCLECI_URL\
+                --header "Circle-Token: $TOBIKO_PRIVATE_CIRCLECI_KEY" \
+                --header "content-type: application/json" \
+                --data '{"branch":"trigger_workflow_on_post","parameters":{"run_main_pr":false, "run_sqlmesh_commit":true, "sqlmesh_version":"'"$CIRCLE_BRANCH"'"}}'
 
 workflows:
   main_pr:
     jobs:
+      - trigger_private_tests
```

Comment:
Can we move this to the end to prevent unnecessary triggers? If the PR's tests fail, no need to trigger.